x-pack/plugins/ux/common/agent_name.ts

```diff
@@ -0,0 +1,20 @@
+/*
```

Comment:
any chance we can move this file to observability or keep it in APM and have the UX app import it?